transforms/class.js

```diff
@@ -12,6 +12,57 @@
 
 module.exports = (file, api, options) => {
   const j = api.jscodeshift;
+  const util = require('util');
+
+  // Coursera specific sort order, taken from our eslintrc
+  const COMPONENT_METHOD_ORDER = [
```

Comment:
let's try and get the component method sorting upstream sometime: https://github.com/reactjs/react-codemod/pull/21